pkg/kratos/service.go

```diff
@@ -897,21 +933,52 @@ func (s *Service) HasNotEnoughLookupSecretsLeft(ctx context.Context, id string)
 
 func (s *Service) is1FAMethod(method string) bool {
 	switch method {
-	case "password", "webauthn", "oidc":
+	case "password", "oidc":
 		return true
+	case "webauthn":
+		return !s.oidcWebAuthnSequencingEnabled
 	default:
 		return false
 	}
 }
 
-func NewService(kratos KratosClientInterface, kratosAdmin KratosAdminClientInterface, hydra HydraClientInterface, authzClient AuthorizerInterface, tracer tracing.TracingInterface, monitor monitoring.MonitorInterface, logger logging.LoggerInterface) *Service {
+func (s *Service) hydrateKratosLoginFlow(ctx context.Context, flow *kClient.LoginFlow) error {
```

Comment:
suggestion: this method should only do the hydration of the login flow. The `if` check should be done outside of this method, in the calling function so that
```go
  if s.oidcWebAuthnSequencingEnabled {
    s.hydrateKratosLoginFlow( ... )
  }

```
Also, it would make the method simpler and more predictable in "what it does"